xbmc/guilib/test/TestGUIControlFactory.cpp

```diff
@@ -0,0 +1,665 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
```

Comment:
This should probably be `Copyright (C) 2024 Team Kodi` to tell readers when the file was introduced. Unless you're ready to tell the world about that time machine you've been hiding ;)

Reply:
fixed.